### PR TITLE
MBMS-69224 Vets Api  Updated vets-json-schema version

### PR DESCRIPTION
**Pull request by ConnorJeff**
## Summary

- Updating the vets-json-schema version with the most current version

## Related issue(s)

- Vets-json-schema PR: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/955
- Jira Story: https://jira.devops.va.gov/browse/MBMS-69224
- Vets-website PR: https://github.com/department-of-veterans-affairs/vets-website/pull/32895

### CODE REVIEW

**Review by aherzberg — DISMISSED** [review]
LGTM for mobile changes

File: spec/models/form_profile_spec.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
   before do
     stub_evss_pciu(user)
+    user.instance_variable_set(:@ssn, user.ssn.delete('-'))
```
Comment (RachalCassity) [inline spec/models/form_profile_spec.rb:15]: What if the user or ssn is null? Will that cause any issues?

Comment (ConnorJeff) [reply spec/models/form_profile_spec.rb:15]: Yep good call

Comment (ConnorJeff) [reply spec/models/form_profile_spec.rb]: I added that line to resolve test failures, but the solution was elsewhere, so I am removing that addition

**Review by aherzberg — APPROVED** [review]
LGTM

**Review by RachalCassity — DISMISSED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Review by aherzberg — DISMISSED** [review]
LGTM

**Review by ksantiagoBAH — DISMISSED** [review]

**Review by aherzberg — APPROVED** [review]
LGTM

**Comment by rjohnson2011** [comment]
Received the same test errors when running the test suite locally on this branch.

**Review by sonntag-adam — APPROVED** [review]

**Review by ksantiagoBAH — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**